mhs/common/mhs_common/workflow/asynchronous_express.py

```diff
@@ -50,10 +50,12 @@ def __init__(self, party_key: str = None, persistence_store: persistence_adaptor
                                       correlation_id: str,
                                       interaction_details: dict,
                                       payload: str,
-                                      wdo: Optional[wd.WorkDescription])\
+                                      wdo: Optional[wd.WorkDescription]) \
             -> Tuple[int, str, Optional[wd.WorkDescription]]:
 
         logger.info('0001', 'Entered async express workflow to handle outbound message')
+        logger.audit('0100', 'Async-Express outbound workflow invoked.', {})
```

Comment:
Do you really need to pass an empty dict here, can the default within the method be an empty dict?